spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -152,6 +152,34 @@ def self.inspect
         }.to fail_with(/MyClass/)
       end
 
+      it "formats an error message when inspect fails" do
+        klass = Struct.new(:name) do
+          def inspect
+            "<Inspector(#{name})>"
+          end
+        end
+
+        object = klass.new('foo')
+
+        expect(object).to receive(:name).once.and_return('bar')
+
```

Comment:
Remove ws similar to spec above.

Reply:
I _think_ I made the changes you're expecting here, but please let me know if not.